ynr/apps/sopn_parsing/helpers/pdf_helpers.py

```diff
@@ -46,6 +46,18 @@ def document_heading_set(self):
         """
         return self.pages[1].get_page_heading_set()
 
+    @property
+    def blank_doc(self):
+        """
+        Takes the heading from the first or second page (if any) in the document and checks if empty
+        """
+        if len(self.document_heading_set) < 10:
+            if (
+                len(self.pages) > 1
+                and len(self.pages[2].get_page_heading_set()) < 10
+            ):
+                return True
+
```

Comment:
This is looking good now, but to avoid the nested `if` statement I would stucture it differently to return early when the first page has more than 10 words, and then return the return value of your second conditional:
```
    def blank_doc(self):
        """
        Takes the heading from the first or second page (if any) in the document and checks if empty
        """
        if len(self.document_heading_set) >= 10:
            return False

        return (
            len(self.pages) > 1
            and len(self.pages[2].get_page_heading_set()) < 10
        )
```